src/components/Select/Select.tsx

```diff
@@ -18,7 +18,7 @@ import FieldNote from '../FieldNote';
 import Icon, { type IconName } from '../Icon';
 import {
   defaultPopoverModifiers,
-  PopoverContainerV2 as PopoverContainer,
+  default as PopoverContainer,
```

Comment:
a little leftover piece from the 2.0 migration. Tidying the import here so that the major change later is smoother